src/app/disclaimer/disclaimer.component.html

```diff
@@ -0,0 +1,15 @@
+<expansion-panel class="tce-flex tce-justify-between tce-gap-2 tce-flex-col tce-pb-4">
+  <div header>
+    <h2 class="tce-text-xl">Disclaimer</h2>
+  </div>
+  <div content>
+    <p class="tce-text-sm">
+      This tool is provided free of charge to assist you in identifying and measuring your carbon footprint, for
+      information and guidance only. The scores and information provided should not be used for carbon reporting and are
+      made available on an “as is” basis. Scott Logic Limited accepts no liability or responsibility for any use made by
+      any person or organisation of the information or calculations available on this website. Any reliance placed on
+      this website is taken at your own risk. You should read our full website terms and conditions before using this
```

Comment:
I think it would be good to have a link to the full terms and conditions here.

Reply:
Would we link to the Scott Logic terms and conditions there? There was some confusion over this in the legal meeting as it could be used in two places, the TCS site doesn't have terms and conditions yet.

Reply:
I'm not sure but I think we need to check that and provide a link. Also, if we're publishing the tool as an NPM package then this could be on any website, not just one of ours so we need to consider that case as well. Is the idea that the disclaimer will still be on the site for people using this as an NPM package or will we just put a disclaimer in the README for users of the package?

Reply:
Might need to look into this some more as www.scottlogic.com doesn't appear to have terms and conditions either.

Reply:
Updated to link back to `TERMS.md`

Reply:
Looks good!